src/dlgIRC.cpp

```diff
@@ -823,6 +840,11 @@ QPair<bool, QString> dlgIRC::writeIrcHostPort(Host* pH, int port)
     return pH->writeProfileData(dlgIRC::HostPortCfgItem, QString::number(port));
 }
 
+QPair<bool, QString> dlgIRC::writeIrcHostSecure(Host* pH, bool secure)
+{
+    return pH->writeProfileData(dlgIRC::HostSecureCfgItem, QString::number(secure));
```

Comment:
TBH As this is a `bool` value I would *write* the strings `"true"` or `"false"` and use something like the following to process the *read* value (`text`): `text.contains(QLatin1String("true"), Qt::CaseInsensitive)` so that there is no need to decode numeric values if hand editing the setting outside of Mudlet...